diff --git a/node_modules/@uiw/react-markdown-preview/README.md b/node_modules/@uiw/react-markdown-preview/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/dist/markdown.css b/node_modules/@uiw/react-markdown-preview/dist/markdown.css
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/dist/markdown.js b/node_modules/@uiw/react-markdown-preview/dist/markdown.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/dist/markdown.min.css b/node_modules/@uiw/react-markdown-preview/dist/markdown.min.css
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/dist/markdown.min.js b/node_modules/@uiw/react-markdown-preview/dist/markdown.min.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/dist/markdown.min.js.LICENSE.txt b/node_modules/@uiw/react-markdown-preview/dist/markdown.min.js.LICENSE.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/Props.d.ts b/node_modules/@uiw/react-markdown-preview/esm/Props.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/Props.js b/node_modules/@uiw/react-markdown-preview/esm/Props.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/index.d.ts b/node_modules/@uiw/react-markdown-preview/esm/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/index.js b/node_modules/@uiw/react-markdown-preview/esm/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/nodes/copy.d.ts b/node_modules/@uiw/react-markdown-preview/esm/nodes/copy.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/nodes/copy.js b/node_modules/@uiw/react-markdown-preview/esm/nodes/copy.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/nodes/octiconLink.d.ts b/node_modules/@uiw/react-markdown-preview/esm/nodes/octiconLink.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/nodes/octiconLink.js b/node_modules/@uiw/react-markdown-preview/esm/nodes/octiconLink.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/nohighlight.d.ts b/node_modules/@uiw/react-markdown-preview/esm/nohighlight.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/nohighlight.js b/node_modules/@uiw/react-markdown-preview/esm/nohighlight.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/plugins/reservedMeta.d.ts b/node_modules/@uiw/react-markdown-preview/esm/plugins/reservedMeta.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/plugins/reservedMeta.js b/node_modules/@uiw/react-markdown-preview/esm/plugins/reservedMeta.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/plugins/retrieveMeta.d.ts b/node_modules/@uiw/react-markdown-preview/esm/plugins/retrieveMeta.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/plugins/retrieveMeta.js b/node_modules/@uiw/react-markdown-preview/esm/plugins/retrieveMeta.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/plugins/useCopied.d.ts b/node_modules/@uiw/react-markdown-preview/esm/plugins/useCopied.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/plugins/useCopied.js b/node_modules/@uiw/react-markdown-preview/esm/plugins/useCopied.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/preview.d.ts b/node_modules/@uiw/react-markdown-preview/esm/preview.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/preview.js b/node_modules/@uiw/react-markdown-preview/esm/preview.js
old mode 100644
new mode 100755
index 77be2d1..a302ea8
--- a/node_modules/@uiw/react-markdown-preview/esm/preview.js
+++ b/node_modules/@uiw/react-markdown-preview/esm/preview.js
@@ -59,9 +59,9 @@ export default /*#__PURE__*/React.forwardRef((props, ref) => {
     style: style,
     children: /*#__PURE__*/_jsx(ReactMarkdown, _extends({}, customProps, other, {
       skipHtml: skipHtml,
-      urlTransform: urlTransform || defaultUrlTransform,
-      rehypePlugins: pluginsFilter ? pluginsFilter('rehype', rehypePlugins) : rehypePlugins,
-      remarkPlugins: pluginsFilter ? pluginsFilter('remark', remarkPlugins) : remarkPlugins,
+      // urlTransform: urlTransform || defaultUrlTransform,
+      // rehypePlugins: pluginsFilter ? pluginsFilter('rehype', rehypePlugins) : rehypePlugins,
+      // remarkPlugins: pluginsFilter ? pluginsFilter('remark', remarkPlugins) : remarkPlugins,
       children: source || ''
     }))
   }));
diff --git a/node_modules/@uiw/react-markdown-preview/esm/rehypePlugins.d.ts b/node_modules/@uiw/react-markdown-preview/esm/rehypePlugins.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/rehypePlugins.js b/node_modules/@uiw/react-markdown-preview/esm/rehypePlugins.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/styles/markdown.css b/node_modules/@uiw/react-markdown-preview/esm/styles/markdown.css
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/esm/styles/markdown.less b/node_modules/@uiw/react-markdown-preview/esm/styles/markdown.less
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/Props.d.ts b/node_modules/@uiw/react-markdown-preview/lib/Props.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/Props.js b/node_modules/@uiw/react-markdown-preview/lib/Props.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/index.d.ts b/node_modules/@uiw/react-markdown-preview/lib/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/index.js b/node_modules/@uiw/react-markdown-preview/lib/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/nodes/copy.d.ts b/node_modules/@uiw/react-markdown-preview/lib/nodes/copy.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/nodes/copy.js b/node_modules/@uiw/react-markdown-preview/lib/nodes/copy.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/nodes/octiconLink.d.ts b/node_modules/@uiw/react-markdown-preview/lib/nodes/octiconLink.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/nodes/octiconLink.js b/node_modules/@uiw/react-markdown-preview/lib/nodes/octiconLink.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/nohighlight.d.ts b/node_modules/@uiw/react-markdown-preview/lib/nohighlight.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/nohighlight.js b/node_modules/@uiw/react-markdown-preview/lib/nohighlight.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/plugins/reservedMeta.d.ts b/node_modules/@uiw/react-markdown-preview/lib/plugins/reservedMeta.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/plugins/reservedMeta.js b/node_modules/@uiw/react-markdown-preview/lib/plugins/reservedMeta.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/plugins/retrieveMeta.d.ts b/node_modules/@uiw/react-markdown-preview/lib/plugins/retrieveMeta.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/plugins/retrieveMeta.js b/node_modules/@uiw/react-markdown-preview/lib/plugins/retrieveMeta.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/plugins/useCopied.d.ts b/node_modules/@uiw/react-markdown-preview/lib/plugins/useCopied.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/plugins/useCopied.js b/node_modules/@uiw/react-markdown-preview/lib/plugins/useCopied.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/preview.d.ts b/node_modules/@uiw/react-markdown-preview/lib/preview.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/preview.js b/node_modules/@uiw/react-markdown-preview/lib/preview.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/rehypePlugins.d.ts b/node_modules/@uiw/react-markdown-preview/lib/rehypePlugins.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/rehypePlugins.js b/node_modules/@uiw/react-markdown-preview/lib/rehypePlugins.js
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/lib/styles/markdown.less b/node_modules/@uiw/react-markdown-preview/lib/styles/markdown.less
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/markdown.css b/node_modules/@uiw/react-markdown-preview/markdown.css
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/nohighlight.d.ts b/node_modules/@uiw/react-markdown-preview/nohighlight.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/Props.tsx b/node_modules/@uiw/react-markdown-preview/src/Props.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/index.tsx b/node_modules/@uiw/react-markdown-preview/src/index.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/nodes/copy.ts b/node_modules/@uiw/react-markdown-preview/src/nodes/copy.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/nodes/octiconLink.ts b/node_modules/@uiw/react-markdown-preview/src/nodes/octiconLink.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/nohighlight.tsx b/node_modules/@uiw/react-markdown-preview/src/nohighlight.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/plugins/reservedMeta.ts b/node_modules/@uiw/react-markdown-preview/src/plugins/reservedMeta.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/plugins/retrieveMeta.ts b/node_modules/@uiw/react-markdown-preview/src/plugins/retrieveMeta.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/plugins/useCopied.tsx b/node_modules/@uiw/react-markdown-preview/src/plugins/useCopied.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/preview.tsx b/node_modules/@uiw/react-markdown-preview/src/preview.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/rehypePlugins.tsx b/node_modules/@uiw/react-markdown-preview/src/rehypePlugins.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@uiw/react-markdown-preview/src/styles/markdown.less b/node_modules/@uiw/react-markdown-preview/src/styles/markdown.less
old mode 100644
new mode 100755
